src/datasources/asset/components/AssetQueryEditor.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { AssetDataSource } from "../AssetDataSource"
+import { setupRenderer } from "test/fixtures"
+import { AssetQuery } from "../types"
+import { screen, waitForElementToBeRemoved } from "@testing-library/react"
+import { AssetQueryEditor } from "./AssetQueryEditor"
+import { selectors } from "../../../test/selectors";
+
+const render = setupRenderer(AssetQueryEditor, AssetDataSource);
+
+const workspacesLoaded = () => waitForElementToBeRemoved(screen.getByTestId('Spinner'));
+
+test('renders with query defaults', async () => {
+  render({} as AssetQuery)
+  await workspacesLoaded()
+
+  expect(screen.getByTestId(selectors.components.assetPlugin.workspace)).toHaveTextContent('Any workspace');
+  expect(screen.getByTestId(selectors.components.assetPlugin.system)).toHaveTextContent('Select systems');
+})
```

Comment:
We should add a test like [this one](https://github.com/ni/systemlink-grafana-plugins/blob/0e230a372d46e42326bde813bccf74e24c4ed0a8/src/datasources/tag/components/TagQueryEditor.test.tsx#L22) that verifies that the UI renders with the user's saved query.

Reply:
added test _renders with initial query and updates when user makes changes_